### DISCUSSION TITLE
OutOfMemoryError: Java heap space

### DISCUSSION THREAD

**Discussion by SzZuziak** [DependencyTrack/dependency-track#1705] [Q&A]
Hi,
I'm using Dependency Track in my company CI/CD since few month, but from last 2-3 weeks app is getting OutOfMemoryError: Java heap space day by day. I've added 30 RAM to server but the problem still is. Could you please advice what to do with that performance problem?

I'm using the latest version of front and API + jenkins plugin (synchronous mode)

**Comment by nscuro** [comment]
Can you provide more detail of your deployment?

* Are you deploying with Docker Compose / Kubernetes / ...?
* What are the specs of the server you're running the API server on (available RAM, CPU cores)?

> I've added 30 RAM to server

Can you clarify what exactly you did? Did you assign more RAM to your VM, or did you just increase the resource limit in your `docker-compose.yml`?
